{
    "dataType": "CVE_RECORD",
    "cveMetadata": {
        "state": "PUBLISHED",
        "cveId": "CVE-2025-32414",
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "dateUpdated": "2025-04-09T14:57:02.635Z",
        "dateReserved": "2025-04-08T00:00:00.000Z",
        "datePublished": "2025-04-08T00:00:00.000Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unaffected",
                    "product": "libxml2",
                    "vendor": "xmlsoft",
                    "versions": [
                        {
                            "lessThan": "2.13.8",
                            "status": "affected",
                            "version": "0",
                            "versionType": "semver"
                        },
                        {
                            "lessThan": "2.14.2",
                            "status": "affected",
                            "version": "2.14.0",
                            "versionType": "semver"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "In libxml2 before 2.13.8 and 2.14.x before 2.14.2, out-of-bounds memory access can occur in the Python API (Python bindings) because of an incorrect return value. This occurs in xmlPythonFileRead and xmlPythonFileReadRaw because of a difference between bytes and characters."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-393",
                            "description": "CWE-393 Return of Wrong Status Code",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre",
                "dateUpdated": "2025-04-08T02:55:58.812Z"
            },
            "references": [
                {
                    "url": "https://gitlab.gnome.org/GNOME/libxml2/-/issues/889"
                }
            ],
            "x_generator": {
                "engine": "enrichogram 0.0.1"
            },
            "metrics": [
                {
                    "cvssV3_1": {
                        "version": "3.1",
                        "baseScore": 5.6,
                        "baseSeverity": "MEDIUM",
                        "vectorString": "CVSS:3.1/AV:L/AC:H/PR:N/UI:N/S:C/C:L/I:L/A:L"
                    }
                }
            ],
            "cpeApplicability": [
                {
                    "nodes": [
                        {
                            "operator": "OR",
                            "negate": false,
                            "cpeMatch": [
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:xmlsoft:libxml2:*:*:*:*:*:*:*:*",
                                    "versionEndExcluding": "2.13.8"
                                },
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:xmlsoft:libxml2:*:*:*:*:*:*:*:*",
                                    "versionStartIncluding": "2.14.0",
                                    "versionEndExcluding": "2.14.2"
                                }
                            ]
                        }
                    ]
                }
            ]
        },
        "adp": [
            {
                "references": [
                    {
                        "url": "https://gitlab.gnome.org/GNOME/libxml2/-/issues/889",
                        "tags": [
                            "exploit"
                        ]
                    }
                ],
                "metrics": [
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "timestamp": "2025-04-09T14:56:33.455181Z",
                                "id": "CVE-2025-32414",
                                "options": [
                                    {
                                        "Exploitation": "poc"
                                    },
                                    {
                                        "Automatable": "no"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "role": "CISA Coordinator",
                                "version": "2.0.3"
                            }
                        }
                    }
                ],
                "title": "CISA ADP Vulnrichment",
                "providerMetadata": {
                    "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
                    "shortName": "CISA-ADP",
                    "dateUpdated": "2025-04-09T14:57:02.635Z"
                }
            }
        ]
    },
    "dataVersion": "5.1"
}